core/kazoo_endpoint/test/kz_formatters_test.erl

```diff
@@ -32,28 +32,28 @@
 
 regex_inbound_test_() ->
     Formatter = kz_json:from_list([{<<"to">>
-                                   ,[kz_json:from_list([{<<"regex">>, <<"^\\+?1?(\\d{10})$">>}
+                                   ,[kz_json:from_list([{<<"regex">>, <<"^\\+?1?(\\d{10})\$">>}
```

Comment:
@fenollp I know the CI checks failed due to the number manager test, but that means I can't tell if all the \$ changes are actually required or accidental...